src/ArrayFigures.java

```diff
@@ -0,0 +1,24 @@
+public class ArrayFigures {
+
+    private Figures[] list = new Figures[10];
```

Comment:
What if you need more, than 10 figures? What if you need only 1 figure? What if you need to delete figure and etc. In 90% cases it's better to use Collections